pkg/templates/node_1.20.go

```diff
@@ -7,10 +7,14 @@ passwd:
   users:
     - name:          core
       password_hash: {{ .LoginPassword }}
+      groups: [ rkt ]
```

Comment:
This does not work. its stripped out of the transpiled gobbeldigock that gets put into user-data:

```
  "passwd": {
    "users": [
      {
        "name": "core",
        "passwordHash": "$6$rounds=1000000$EHQwRTtoqLxuJNOM$/sU35OnW0cobfq1P8I6qQ/2DMqUNdpeYVzJjX8/NCPMkG.Q.F7lK0MfDRQWOgbE.jaFYzXsJVQwQGjbn/c3G11"
      }
    ],
    "groups": [
      {
        "name": "rkt",
        "system": true
      }
    ]
  }
```